### DISCUSSION TITLE
prost_build config to get protobuf enums serializing through serde as their string names?

### DISCUSSION THREAD

**Discussion by jakswa** [tokio-rs/prost#883] [Q&A]
I'm working with this protobuf definition right now: https://developers.google.com/static/transit/gtfs-realtime/gtfs-realtime.proto

Right now this enum:
```protobuf
enum OccupancyStatus {
    // The vehicle is considered empty by most measures, and has few or no
    // passengers onboard, but is still accepting passengers.
    EMPTY = 0;

    // The vehicle has a relatively large percentage of seats available.
    // What percentage of free seats out of the total seats available is to be
    // considered large enough to fall into this category is determined at the
    // discretion of the producer.
    MANY_SEATS_AVAILABLE = 1;
}
optional OccupancyStatus occupancy_status = 9;
```

Is showing up in my JSON as:
```json
"occupancy_status": 1
```

Is there a straightforward way to get this value showing up as `MANY_SEATS_AVAILABLE`? Or even `ManySeatsAvailable` (which I see in the generated prost rust)?

**Comment by jakswa** [comment]
I forgot my current `build.rs` for prost:
```rust
prost_build::Config::new()
    .message_attribute(".", "#[derive(serde::Serialize)]")
    .compile_protos(&["src/gtfs-realtime.proto"], &["src/"])?;
```

edit: and I'm doing JSON serialization way above on a parent struct several levels higher. I know there's `as_str_name` but I haven't figured out how to convert the i32 using `OccupancyStatus::from_i32` without building a custom JSON struct to serialize...

**Comment by BrennerSpear** [comment]
having the same issue with `prost_build`, when using sqlx. I'd like to be able to have the enums be generated without Explicit Discriminants 

I want
```
enum OccupancyStatus {
    EMPTY,
    MANY_SEATS_AVAILABLE,
}
```
not
```
 enum OccupancyStatus {
    EMPTY = 0;
    MANY_SEATS_AVAILABLE = 1;
}
```
